RELEASE_TEMPLATE.md

```diff
@@ -37,38 +37,52 @@ Times are relative to the start of the release.
 - 00:25 Perform the Validation Checklist steps below.
 - 01:00 Activity complete. Change the release status to "Complete".
 
-## Activity Checklist
-Refer to the Validation Activities section for more details about each release step.
+Refer to the Validation Activities section for more details about each activity and validation checklist step.
 
+## Activity Checklist
 - [ ] Prereqs to be performed prior to the release starting:
+- [ ] Create a release-$$blox_version$$ branch off of the dev branch in your Github fork.
 - [ ] Ensure that the cluster-state-service unit-tests pass.
 - [ ] Ensure that the cluster-state-service end-to-end tests pass.
 - [ ] Ensure that the daemon-scheduler unit-tests pass.
 - [ ] Ensure that the daemon-scheduler end-to-end tests pass.
 - [ ] Ensure that the daemon-scheduler integration tests pass.
-- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct image versions.
-- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct image versions.
-- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions.
-- [ ] Ensure that a $$blox_version$$ change log entry exists in `CHANGELOG.md`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/README.md` and `daemon-scheduler/README.md`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/VERSION` and `daemon-scheduler/VERSION`.
-- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/versioning/version.go` and `daemon-scheduler/versioning/version.go`.
-- [ ] Ensure that the technician has a GitHub account and permissions to merge into the Blox master branch.
-- [ ] Ensure that the technician has a Docker Hub account and permissions to push into the bloxoss repositories.
+- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct image versions in the release-$$blox_version$$ branch.
+- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions in the release-$$blox_version$$ branch.
```

Comment:
The only one that should ever change is `IamRoleTask`. I've updated the description and added a section with more details.